Mod2/Lessons/Week1/SQLCRUD.md

```diff
@@ -0,0 +1,132 @@
+
+## Updating Records
+
+Occasionally, we will need to update records.  We can do that with the `UPDATE` command.
+
+> With a partner, take a look at the update statement below and answer the following questions:
+> * What do we need to know about a table in order to update a record?
+> * What do you think would happen if we tried to update a record that doesn't exist?
+> * What do you think would happen if we did not include a **WHERE** in our statement?
+>
+> Be ready to share out!
+> 
+> ![](/Mod2/Images/Week1/SQL_UPDATE.png)
+
+** Instructor Note ** Be sure to point out that if we forget the `WHERE`, all records in the table will be updated!
+
+### Practice UPDATE
+
+> With a partner, use what you know about UPDATE to make changes to one or more records in the orders table.  
+> As an extra challenge:
+> * Update more than one column with one statement.
+> * Update more 2 or more records with one statement.
+> Do some googling and/or use [this resource](https://www.w3schools.com/sql/sql_update.asp) to help with these challenges!
+>
+> Reach out to an instructor with any questions or issues that come up!
+
+## Filtering Records
+
+So far, we have manipulated records in the database (with INSERT, UPDATE, and DELETE), and we have seen the current state of tables (with SELECT *).
+
+Seeing the state of an entire table is great, and with a small dataset, it might be all we need.  But, when we start working with databases that contain many tables, with millions of records, we will need to be more specific when SELECTing our data.
+
+> With a partner, take a look at the select statements below and answer the following questions:
+> * What do we need to know about a table in order to write a good select statement?
+> * In your own words, what is each statement doing?
+> * Given these examples, what does `*` indicate?
+> 
+> Be ready to share out!
+> 
+> ![](/Mod2/Images/Week1/SQL_SELECT.png)
+
+### Practice SELECT
+
+> With a partner, use what you know about SELECT and WHERE to create statements that would return the following from the orders table:
+> * All columns where the quantity ordered is greater than 2
+> * The Price and Item for all of Naji Park's order
+> * The Items that cost less than $20 (2000 cents)
+> 
+> Reach out to an instructor with any questions!
+
+### More SELECTing
+
+There are so many things we can do by combining SELECT with other commands.  We have seen how we can use SELECT and WHERE together; take a look at the examples below for some more ways of SELECTing.  Feel free to try these out on your tables to explore what they do!
+
+![](/Mod2/Images/Week1/SQL_ORDERBY.png)
+
+## Check for Understanding
+* Imagine you are in an interview.  The interviewer asks: How would you describe CRUD? (make sure to use examples!)
+* Why do we need to use a `WHERE` when deleting or updating records?
+* Match each CRUD action to one (or more) SQL commands:
+|CRUD Action|SQL Command(s)|
+|||
+|||
+|||
```

Comment:
I really like how you've structured this lesson with more of a looking at code and predicting what happens approach than an instructor lecturing approach!

My only overall comment is that I think throughout we should encourage students to try things and see what happens throughout. Might even be good to have an instructor note to reassure students that they are not going to break their computer, they are just interacting with their practice LittleShop database.